handlers/deck_handler.py

```diff
@@ -0,0 +1,74 @@
+from hearthstone import deckstrings
+import binascii
+import requests
+import json
+
+class DeckHandler():
+
+    def __init__(self, deck_response):
+        self.deck_response = deck_response
+        self.decks = self.get_decks()

```

Comment:
This should probably be a cached property with a timeout of 1 hours (we refresh every 1-3 currently). Otherwise the data will get stale once HearthBot has been running for a few days.